### PR TITLE
pwa icons are added and gitignore is updated

### PR DESCRIPTION
**Pull request by ssantss**
pwa icons are added and gitignore is updated

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,21 @@
...
+    <link rel="apple-touch-icon"  href="/images/iOS/touch-icon-ipad-retina.png" sizes="152x152">
+    <meta name="apple-mobile-web-app-title" content="Abir en whatsapp">  
+    <!--ICONOS DE ANDROID-->
+    <link rel="icon" href="/images/Android/Icon-72.png" sizes="72x72">
+    <link rel="icon" href="/images/Android/Icon-96.png" sizes="96x96">
+    <link rel="icon" href="/images/Android/Icon-144.png" sizes="144x144">
+    <link rel="icon" href="/images/Android/Icon-192.png" sizes="192x192">
+    <!--QUITAR BARRAS IOS-->
+    <meta name="apple-mobile-web-app-capable" content="yes"> 
+    <!--QUITAR BARRAS ANDROID-->
```
Comment (maoaiz) [inline index.html]: Can you remove the spanish comments please? or type them in english

File: manifest.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+
+    "short_name": "Open in Whatsapp",
+    "name": "Abrir numero en Whatsapp",
```
Comment (maoaiz) [inline manifest.json]: Please type the name in english too.

**Review by maoaiz — CHANGES_REQUESTED** [review]
Thanks for your contributions, please check te comments and let me know when you are ready

**Comment by ssantss** [comment]
Se realizaron todos los cambios pedidos!

File: index.html [hunk 3]
```diff
@@ -3,6 +3,19 @@
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <!--iOS-->
+    <link rel="apple-touch-icon" href="/images/iOS/touch-icon-iphone.png" >
+    <link rel="apple-touch-icon"  href="/images/iOS/touch-icon-ipad.png" sizes="76x76">
+    <link rel="apple-touch-icon"  href="/images/iOS/touch-icon-iphone-retina.png" sizes="120x120">
+    <link rel="apple-touch-icon"  href="/images/iOS/touch-icon-ipad-retina.png" sizes="152x152">
+    <meta name="apple-mobile-web-app-title" content="Abir en whatsapp">
```
Comment (maoaiz) [inline index.html]: typo

**Review by maoaiz — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (ssantss) [reply index.html]: Ese es el nombre de la apliacion cuando se agrega al inicio! ud prefiere que esta se muestre en ingles?

Comment (maoaiz) [reply index.html]: English please

Comment (maoaiz) [reply index.html]: I mean you have a typo here, it says "Abir" instead of "Abrir"

Comment (ssantss) [reply index.html]: ok, do you prefer the text in english or spanish?

Comment (maoaiz) [reply index.html]: Spanish is ok for now

Comment (ssantss) [reply index.html]: ok

File: index.html [hunk 4]
```diff
@@ -3,6 +3,19 @@
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <!--iOS-->
+    <link rel="apple-touch-icon" href="/images/iOS/touch-icon-iphone.png" >
+    <link rel="apple-touch-icon"  href="/images/iOS/touch-icon-ipad.png" sizes="76x76">
+    <link rel="apple-touch-icon"  href="/images/iOS/touch-icon-iphone-retina.png" sizes="120x120">
+    <link rel="apple-touch-icon"  href="/images/iOS/touch-icon-ipad-retina.png" sizes="152x152">
+    <meta name="apple-mobile-web-app-title" content="Open in whatsapp">
```
Comment (maoaiz) [inline index.html]: Why you changed this to english again? It should be spanish.

File: manifest.json (hunk 2 above)
Comment (maoaiz) [reply manifest.json]: Please add the accent mark in the word "número" or use a different sentense like "Abrir chat de Whatsapp"

**Review by maoaiz — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (ssantss) [reply index.html]: ok

**Review by maoaiz — APPROVED** [review]
Thanks man

**Pull request merged**